i18n/langES-obj.js

```diff
@@ -41,6 +41,22 @@
 
 
 var localeObj = {
+    time: {
+        month: [
+            'Enero', 'Febrero', 'Marzo', 'Abril', 'Mayo', 'Junio',
```

Comment:
Spanish doesn't capitalize on month or day-of-week names. So it should be `'enero'` and etc.

Reply:
Done. Knowledge is limited but helps a lot. :)